database/migrations/2021_12_10_190950_payment_table.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Support\Facades\DB;
+
+class PaymentTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        DB::statement("
+            CREATE TABLE `payment` (
+            `id` bigint(20) AUTO_INCREMENT,
```

Comment:
why not unsigned.

Reply:
As helpers were used in few places. You can ignore it.

Reply:
changed thanks!